kustomize/manager/kustomization.yaml

```diff
@@ -8,4 +8,4 @@ configurations:
 images:
   - name: controller
     newName: ghcr.io/kubecfg/kubit
-    newTag: v0.0.16@sha256:d9e38e935bdb5f084bc9b1f5980987e165f7e16f0bfbef0d24d5664f9bf2c7e2
+    newTag: v0.0.17
```

Comment:
Based on @jdockerty's comment [here](https://github.com/kubecfg/kubit/pull/399#discussion_r1570361263), I believe there is some automation that will inject the new digest here at some point (not clear to me when/how though).